Packs/FeedAzure/Integrations/FeedAzure/FeedAzure_test.py

```diff
@@ -192,15 +197,3 @@ def test_extract_metadata(regions_list, services_list, values_group_section, exp
 def test_extract_indicators(regions_list, services_list, values_group_section, expected_result):
     client = Client(regions_list, services_list)
     assert (client.extract_indicators_from_values_dict(values_group_section) == expected_result)
-
-
-def test_get_indicators(mocker):
-    mocker.patch.object(demisto, 'args', return_value={
-        'regions': 'norwaye, westus',
-        'services': 'AzureBackup, AzureAppService',
-    })
-    mocker.patch.object(demisto, 'command', return_value='azure-get-indicators')
-    mocker.patch.object(demisto, 'results')
-
-    main()
-    assert demisto.results.call_args[0][0]['Contents']
```

Comment:
Why have we deleted this?

Reply:
As stated in the Description - its duplicate with the PB, was written before the decision to have PB for the feed.